gsa/src/web/pages/usersettings/usersettingspage.js

```diff
@@ -319,6 +328,15 @@ class UserSettings extends React.Component {
     if (isLoading) {
       return <Loading/>;
     };
+    const openVasScanConfigs = scanconfigs.filter(
+        conf => conf.scan_config_type === OPENVAS_SCAN_CONFIG_TYPE);
+    const ospScanConfigs = scanconfigs.filter(
+      conf => conf.scna_config_type === OSP_SCAN_CONFIG_TYPE);
```

Comment:
We should add some filter functions to models/scanconfig.js for this
```js
const ospScanConfigsFilter = config => config.scan_config_type === OSP_SCAN_CONFIG_TYPE;
```
This will hide the access to the scan_config_type property which we are going to rename in future.